yb-voyager/src/query/queryissue/constants.go

```diff
@@ -415,6 +415,25 @@ const (
 	SQL_BODY_IN_FUNCTION_ISSUE_DESCRIPTION                     = "SQL language functions with an inline body are not supported in YugabyteDB."
 	UNIQUE_NULLS_NOT_DISTINCT_ISSUE_DESCRIPTION                = "Unique constraint treating NULL values as non-distinct is not yet supported in YugabyteDB."
 	UNSUPPORTED_EXTENSION_ISSUE_DESCRIPTION                    = "This extension is not supported in YugabyteDB by default. Refer to the docs link for the more information on supported extensions."
+
+	//Hotspot on timestamp/date indexes
+
+	HOTSPOTS_ON_TIMESTAMP_INDEX                       = "HOTSPOTS_ON_TIMESTAMP_INDEX"
+	HOTSPOTS_ON_DATE_INDEX                            = "HOTSPOTS_ON_DATE_INDEX"
+	HOTSPOTS_ON_TIMESTAMP_INDEX_ISSUE                 = "Hotspots with range-sharded timestamp indexes"
+	HOTSPOTS_ON_DATE_INDEX_ISSUE                      = "Hotspots with range-sharded date indexes"
+	HOTSPOTS_ON_RANGE_SHARDED_INDEX_ISSUE_DESCRIPTION = `Indexes on timestamp or date columns can lead to read/write hotspots in distributed databases like YugabyteDB, primarily due to the sequential nature of these values (e.g., inserting recent timestamps). This sequential pattern can cause an uneven distribution of data and query load, leading to performance bottlenecks.
+To address this issue and improve query performance, application-level sharding is recommended. This approach involves adding an additional column to the table and indexing both the new column and the timestamp/date column. The additional column is used to distribute data using a hash-based strategy, effectively spreading the load across multiple nodes.
```

Comment:
shall we refrain from providing the details of the workaround here? We can keep it limited to "application-level sharding is recommended." . The docs link can provide the entire set of details with an example. 

I feel like: explaining the workaround briefly is difficult, so can just mention the workaround, and the docs will have more information anyway.

Reply:
hmm okay, will remove the application level sharing explanation part